stglib/eofe.py

```diff
@@ -0,0 +1,632 @@
+from __future__ import division, print_function
```

Comment:
This line is not needed; it's a holdover from Python 2.

Reply:
Line removed.